test/models/profile_test.rb

```diff
@@ -79,4 +79,33 @@ class ProfileTest < ActiveSupport::TestCase
     profiles(:one).update(business_objective: nil)
     assert_empty BusinessObjective.where(title: 'abcd')
   end
+
+  context 'cloning profile to account' do
+    should 'create host relation when the profile is created' do
+      assert_difference('ProfileHost.count', 1) do
+        cloned_profile = profiles(:one).clone_to(
```

Comment:
Perhaps it's a bit unclear if you don't see the `setup` method in this file - `profiles(:one)` has `hosts(:one)`, but not `hosts(:two)`

Reply:
Asserting that `host(:two)` is not yet included before `clone_to` would make that more obvious.